### ISSUE TITLE
Tensorflow target tensor error

### ISSUE THREAD

**Issue by Orzelius** [Orzelius/Hiragata_ai#1]
The current [model](https://github.com/Orzelius/Hiragata_ai/blob/main/src/model.ts#L51-L77) is throwing error
```
Error when checking target: expected dense_Dense2 to have shape [,1], but got array with shape [80841,46]
```
The issue lies in the labels I'm passing to the `model.fit` function [here](https://github.com/Orzelius/Hiragata_ai/blob/main/src/model.ts#L117).

The labels tensor is created [here](https://github.com/Orzelius/Hiragata_ai/blob/main/src/data/loadData.ts#L102) and is of shape `[num_classes, 46]`

The model layers look like this:
```
_________________________________________________________________
Layer (type)                 Output shape              Param #   
=================================================================
conv2d_Conv2D1 (Conv2D)      [null,46,46,32]           320
_________________________________________________________________
max_pooling2d_MaxPooling2D1  [null,23,23,32]           0
_________________________________________________________________
conv2d_Conv2D2 (Conv2D)      [null,21,21,64]           18496
_________________________________________________________________
max_pooling2d_MaxPooling2D2  [null,10,10,64]           0
_________________________________________________________________
conv2d_Conv2D3 (Conv2D)      [null,8,8,64]             36928
_________________________________________________________________
max_pooling2d_MaxPooling2D3  [null,4,4,64]             0
_________________________________________________________________
flatten_Flatten1 (Flatten)   [null,1024]               0
_________________________________________________________________
dropout_Dropout1 (Dropout)   [null,1024]               0
_________________________________________________________________
dense_Dense1 (Dense)         [null,512]                524800
_________________________________________________________________
dense_Dense2 (Dense)         [null,46]                 23598
=================================================================
Total params: 604142
Trainable params: 604142
Non-trainable params: 0
```

Whi is the last `softmax` layer ([it's here](https://github.com/Orzelius/Hiragata_ai/blob/main/src/model.ts#L64)) not applying the correct labels shape?
